packages/itinerary-body/src/util.ts

```diff
@@ -174,3 +174,8 @@ export function getPlaceName(
   // Default to place name
   return place.name;
 }
+
+// TODO: is this the best way to do this?
```

Comment:
is there a `date-fns` function that could do this for us?

Reply:
I looked and failed